how-tos/testing-your-backend.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Testing Your Backend"
+
+---
+
+When your frontend makes a request to your backend, it [includes an access token](https://docs.propelauth.com/how-tos/making-authenticated-requests). But, what if we don’t have a frontend yet? Or what if we want to test our backend independently from our frontend?
+
+The following guide will walk you through how to use and test the PropelAuth backend libraries in isolation from your frontend.
+
+## Getting an Access Token Without a Frontend
+
+There are a few different ways that you can get started with initializing this process, but they all boil down to creating an [Access Token](https://docs.propelauth.com/overview/access-tokens) for use in your testing.
```

Comment:
You can use relative URLs for these so local development will link to the right place